Assets/Scripts/Player/PlayerController.cs

```diff
@@ -60,7 +60,8 @@ public class PlayerController : MonoBehaviour {
 	}
 
 	private void Update() {
-		isGrounded = CheckIfGrounded();
+		isGrounded = groundedCollider.IsTouchingLayers(groundLayer);
+
```

Comment:
Move this to `FixedUpdate`. Physics is not updated every frame so checking this in `Update` is unnecessary.

Reply:
Ye, though about doing the check in triggerEnter/Exit instead.

Reply:
That would also work.